manager/controlapi/service.go

```diff
@@ -645,6 +651,8 @@ func (s *Server) checkConfigExistence(tx store.Tx, spec *api.ServiceSpec) error
 // - Returns `AlreadyExists` if the ServiceID conflicts.
 // - Returns an error if the creation fails.
 func (s *Server) CreateService(ctx context.Context, request *api.CreateServiceRequest) (*api.CreateServiceResponse, error) {
+	// TODO(gianarb) not sure about how to get here the nodeID or something
+	// similar. but the service has already a host at this point?
```

Comment:
The service is not associated with any particular nodes when it is first created.